deploy/basic/README_MSZ.md

```diff
@@ -21,10 +21,37 @@ If using local Terraform or on CloudShell:
+
+![DNS Delegation](../../images/basic/DNS-Delegation.png)
+
+Following this, enable the DNS and certificate parameters within the `terraform.tfvars` file that you have prepared.  The DNS zone and WAF policy will be created within the LB compartment.  The DNS zone you configure for terraform can be a subdomain to the owned domain name, such as `mushop.example.com`. When WAF is enabled, 2 records will be added, one for the internal load balancer, and one for WAF, which users will hit.  The certificate that is generated by Lets Encrypt will use the WAF name as its Common Name and will also add the LB as a Subject Alternative name.
+
+Regarding certificate generation, Lets Encrypt encourages the use of the staging URL by default, and only to use the production URL when ready, as there are limits to daily usage.  The default parameter in the configuration uses the staging URL, so it is recommended to switch it once you are sure everything is working. It is possible to use terraform targeting to test this, but you must be careful to create the DNS zone and certificate together.  If you do not need to do this, simply run through the entire process and switch the `server_url` variable afterwards and re-run.
+
+Another notable change that will occur when using WAF is that a new Network Security Group will be configured and applied directly to the Load Balancer.  This NSG allows access to the LB only from the list of IP addresses associated with Oracle WAF.  Consequently, the Security List for the LB Subnet will not provide access on ports 80 or 443, as the NSG will open 443 only from WAF IP addresses to the Load Balancer.  It is recommended that you change the `dns_lb_entry` variable to something like `internal`, and then set the `dns_waf_entry` to `store`, which will then be accessible externally as `store.mushop.example.com`.
+
+Lastly when enabling WAF, you will experience a longer terraform run time than before - it takes roughly 25 minutes to create a WAF Policy, and only after that will you see the WAF DNS record in the created zone.  In order to allow terraform to run and complete without risking a timeout, you should run it as follows:
+```bash
+prompt> nohup terraform apply -auto-approve >> apply.txt 2>&1 &
+prompt> tail -f apply.txt
+```
+
+Once the terraform completes, the WAF Policy and DNS Zone should reflect correctly, and you can hit the external URL, such as `store.mushop.example.com`.  The browser should indicate a clean certificate, and the information dialog for it should show what is configured.  Finally, if the CAPTCHA challenge for WAF is enabled, test that by navigating the store to any product page, and it will challenge you before displaying content.
```

Comment:
This can potentially break the load test scripts for the load test and auto-scaling labs.

Reply:
I would say that if WAF is used and CAPTCHA is not enabled by default, the load test should be ok.  Let me look at the defaults. CAPTCHA should be the last option to enable.